resources/config_bundles/linux_rooted/patch_order.list

```diff
@@ -1,17 +1,14 @@
-inox-patchset/chromium-vaapi-move.patch
-inox-patchset/chromium-vaapi-init.patch
-inox-patchset/chromium-vaapi-rgbx.patch
+inox-patchset/chromium-vaapi-init-r16.patch
 inox-patchset/chromium-vaapi-r16.patch
 
 debian/master-preferences.patch
 debian/system/icu.patch
 debian/system/vpx.patch
```

Comment:
The vpx and lcms2 patches can also be removed for Chromium 65, see the [Debian series file](https://sources.debian.org/src/chromium-browser/65.0.3325.146-1/debian/patches/series/).

Reply:
You're right, I'll remove them